doc/se/Snabbstart

```diff
@@ -15,11 +15,11 @@ Innehåll:
 
 1.1) Om ScummVM:
 ---- -----------
-ScummVM är ett program som gör det möjligt att spela vissa klassiska peka-och-klicka-äventyrsspel, förutsatt att du redan har de nödvändiga datafilerna. Det finurliga i det hela är att ScummVM ersätter de ursprungliga programfilerna som följde med spelet, vilket låter dig spela dem på operativsystem de aldrig var designade för!
+ScummVM är ett program som gör det möjligt att spela vissa klassiska ”peka-och-klicka”-äventyrsspel, förutsatt att du redan har de nödvändiga datafilerna. Det finurliga i det hela är att ScummVM ersätter de ursprungliga programfilerna som följde med spelet, vilket låter dig spela dem på operativsystem de aldrig var designade för!
```

Comment:
As someone who isn't used to these beautiful and typographically correct quotation marks, why are some of these left quotation marks opening quotation marks and some are closing quotation marks?

Reply:
That's how it was before, and matches the LasMig file. Should I fix both files here or as part of a separate PR?